src/vs/editor/contrib/referenceSearch/referencesWidget.ts

```diff
@@ -695,32 +639,39 @@ export class ReferenceWidget extends PeekViewWidget {
 				accessibilityProvider: new AriaProvider()
 			};
 
+			var options: tree.ITreeOptions = {
+				twistiePixels: 20,
+				ariaLabel: nls.localize('treeAriaLabel', "References"),
+				keyboardSupport: false
+			};
+
+			this._tree = this._instantiationService.createInstance(WorkbenchTree, div.getHTMLElement(), config, options);
+			this._callOnDispose.push(attachListStyler(this._tree, this._themeService));
+
+			ctxReferenceWidgetSearchTreeFocused.bindTo(this._tree.contextKeyService);
+
 			// listen on selection and focus
-			this._disposables.push(controller.onDidFocus((element) => {
+			var onEvent = (element: any, kind: 'show' | 'goto' | 'side') => {
```

Comment:
`var` 😱

Reply:
@jrieken that is consistent with other usages of `var` in `_fillBody`, I did not want to change that style. There are more `vars` throughout the file, you can clean that up afterwards if you want.